R/input-utils.R

```diff
@@ -90,31 +90,32 @@ generateOptions <- function(inputId, selected, inline, type = 'checkbox',
   div(class = "shiny-options-group", options)
 }
 
+# If a list/vector is unnamed, give it blank names
+ensureNamed <- function(x) {
```

Comment:
It turns out we already have a function that does this, so we should use that:
https://github.com/rstudio/shiny/blob/250303790c1121a0f8de7bf929e786696e93ab25/R/utils.R#L176-L183

I like your implementation better than the one that currently exists for `asNamedVector`, though.